backend/src/firmware/boards-infos.json

```diff
@@ -54,7 +98,18 @@
                 "offset": 65536
             }
         ],
-        "board": "esp12e"
+        "board": "esp12e",
+        "defaults": {
+            "imuSDA": "4",
+            "imuSCL": "5",
+            "imuInts": ["14", "13"],
+            "led": "2",
+            "enableLed": true,
+            "ledInverted": false,
```

Comment:
`//  #define LED_INVERTED false`
In <defines.h> this is not defined -> default from <globals.h>
is ledInverted true